cmd/manager/main.go

```diff
@@ -118,6 +120,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	klog.Info("checking old common services if installed.")
+	exist, err := check.CheckOriginalCs(mgr)
+	if err != nil {
+		klog.Error(err)
+		os.Exit(1)
+	}
+	if exist {
+		klog.Error("old common services has been installed, uninstall the old common services before install the new")
```

Comment:
The Helm based IBM Common Services must be uninstalled before performing operator based installation.

Reply:
done